src/sentry/snuba/subscriptions.py

```diff
@@ -133,11 +140,11 @@ def bulk_create_snuba_subscriptions(projects, subscription_type, snuba_query):
     subscriptions = []
     # TODO: Batch this up properly once we care about multi-project rules.
     for project in projects:
-        subscriptions.append(create_snuba_subscription(project, subscription_type, snuba_query))
+        subscriptions.append(create_snuba_subscription(project, subscription_type, alert_rule))
     return subscriptions
 
 
-def create_snuba_subscription(project, subscription_type, snuba_query) -> QuerySubscription:
+def create_snuba_subscription(project, subscription_type, alert_rule) -> QuerySubscription:
```

Comment:
Same feedback - it doesn't seem like we're getting anything out of passing alert_rule here and it's breaking the separation of concerns

Reply:
Ah yea fair.
I initially was going to pass this down because i was going to gate the creation of the query _in_ snuba here.

but as discussed, we'll now gate the entire QuerySubscription instead